lib/gnat.ex

```diff
@@ -247,48 +285,59 @@ defmodule Gnat do
   defp socket_write(%{socket: socket, connection_settings: %{tls: true}}, iodata) do
     :ssl.send(socket, iodata)
   end
+
   defp socket_write(%{socket: socket}, iodata), do: :gen_tcp.send(socket, iodata)
 
-  defp add_subscription_to_state(%{receivers: receivers}=state, sid, pid) do
+  defp add_subscription_to_state(%{receivers: receivers} = state, sid, pid) do
     receivers = Map.put(receivers, sid, %{recipient: pid, unsub_after: :infinity})
     %{state | receivers: receivers}
   end
 
-  defp cleanup_subscription_from_state(%{receivers: receivers}=state, sid, []) do
+  defp cleanup_subscription_from_state(%{receivers: receivers} = state, sid, []) do
     receivers = Map.delete(receivers, sid)
     %{state | receivers: receivers}
   end
-  defp cleanup_subscription_from_state(%{receivers: receivers}=state, sid, [max_messages: n]) do
+
+  defp cleanup_subscription_from_state(%{receivers: receivers} = state, sid, max_messages: n) do
     receivers = put_in(receivers, [sid, :unsub_after], n)
     %{state | receivers: receivers}
   end
 
   defp process_message({:msg, topic, sid, reply_to, body}, state) do
     unless is_nil(state.receivers[sid]) do
-      send state.receivers[sid].recipient, {:msg, %{topic: topic, body: body, reply_to: reply_to, gnat: self()}}
+      send(state.receivers[sid].recipient, {
+        :msg,
+        %{topic: topic, body: body, reply_to: reply_to, gnat: self()}
+      })
```

Comment:
Similar here. Splitting out the map to make the tuple on the same line would be 👌